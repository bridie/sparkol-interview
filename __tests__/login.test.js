import React from 'react';
import renderer from 'react-test-renderer';

import Login from '../src/js/components/login';

it('renders correctly', () => {
  const tree = renderer.create(<Login />).toJSON();
  expect(tree).toMatchSnapshot();
});
