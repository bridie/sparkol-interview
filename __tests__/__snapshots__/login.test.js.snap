// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="l-heading"
  >
    <h1>
      Login
    </h1>
  </div>
  <form
    className="c-login-form"
    onSubmit={[Function]}
  >
    <label
      className="c-login-form__label"
      htmlFor="username"
    >
      Username
      <input
        className="c-login-form__input"
        id="username"
        onChange={[Function]}
        required={true}
        type="text"
        value=""
      />
    </label>
    <label
      className="c-login-form__label"
      htmlFor="password"
    >
      Password
      <input
        className="c-login-form__input"
        id="password"
        onChange={[Function]}
        required={true}
        type="password"
        value=""
      />
    </label>
    <button
      className="c-button"
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
